SHARDING.md

```diff
@@ -0,0 +1,465 @@
+by a healthcheck for HTTP and HTTP/2 traffic.
+
+Under Backend buckets, you should see a storage bucket and Cloud CDN should be Enabled.
+
+### Verify the read path
+
+Verify that a checkpoint is served from the Storage bucket, e.g.
+
+```
+curl https://log2026-1.rekor.sigstage.dev/api/v2/checkpoint
+
+log2026-1.rekor.sigstage.dev
+0
+YR3vjr9qTpYPib0tUJT/7/pAPuK9xDOibMncp2aQAJE=
+
+— log2026-1.rekor.sigstage.dev 8w1amdxKovmc7H9+4lnx/vSeAQo9zo8cr8EHFCVviWMtTmm/Xw+zGDnBni9/mHHa1hFXIQlC29+YyaOXRxduvbE20Qg=
+```
+
+### Verify the write path
+
+Following
+[these instructions](https://github.com/sigstore/rekor-tiles/blob/main/CLIENTS.md#rekor-v2-the-bash-way),
+test the write path for the new shard. This will sign an entry with a private key and upload
+the signing event to the log.
+
+## WIP: Update Prober
+
+Tracked in https://github.com/sigstore/rekor-tiles/issues/46. We're aiming to avoid
+needing to manually update the prober. The prober should pick up new shards from the
+SigningConfig, with the latest active shard having no end date. Only the active
+shard will accept write traffic. All other shards in the SigningConfig can still
+handle read traffic.
+
+We will need a mechanism for running prober tests against a new deployment before we've
+distributed an updated SigningConfig and TrustedRoot. Assuming this is a parameter to
+a GitHub Actions workflow, a deployer should provide the URL to the new shard and run
+prober tests.
+
+## Update TUF Verification Material
+
+Now that the shard is up and we've successfully signed and verified using it, we will
+update the TrustedRoot and SigningConfig files distributed via TUF so that clients
+can verify proofs from the shard and use the shard during signing.
+
+For TrustedRoot, the validity window for the shard should be from the point
+when the shard was spun up. For SigningConfig, the validity window should start
+after the TUF timestamp expires to guarantee all clients will have picked up
+the latest TrustedRoot. In practice for our TUF root with timestamp validity
+of one week, this means setting the validity window to be at least one week
+past when the TUF targets are signed.
+
+You will need two timestamps:
+
+* New shard's start for SigningConfig, which should be one week past when the root-signing PR is merged
+and new TUF repository is published. Overestimate (by an hour or even a day) if needed,
+as you don't want clients to start using the new shard before all clients
+have picked up the latest TrustedRoot.
+* Previous shard's end for TrustedRoot: Clients will consider this shard's entry timestamps invalid
+if they are after this date. This must be after the start of the new shard. It doesn't have to
+overly precise, and can be updated in a future TUF signing event.
+
+### Retrieve shard public key
+
+Look at the service logs to find the public key. This is the easiest way to retrieve
```

Comment:
Add a link to the service logs for an existing shard, so we can easily know how to find the logs for the next shard.

Reply:
The audience for this doc is someone familiar with the infrastructure so they should be familiar with where to retrieve logs for pods.